var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -32,6 +33,8 @@ class SuperluDist(Package):
     homepage = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/"
     url = "http://crd-legacy.lbl.gov/~xiaoye/SuperLU/superlu_dist_4.1.tar.gz"
 
+    version('develop', git='https://github.com/xiaoyeli/superlu_dist', tag='master')
+    version('xsdk-0.1.0', git='https://github.com/xiaoyeli/superlu_dist', commit='0b5369f')
```

Comment:
FYI,  you can't build `petsc@3.7:` with `superlu-dist@xsdk-0.1.0` because `petsc` `depends_on('superlu-dist@5.0.0:', when='@3.7:+superlu-dist+mpi')` whereas `xsdk-0.1.0 < 5.0.0`. 

Workaround is https://github.com/LLNL/spack/pull/1983 + `develop.xsdk.0.1.0` or alike.